Makefile

```diff
@@ -57,6 +56,11 @@ git_export: clean
 	# Windows needs a different name for the startup script - due to process creation
 	# (no fork/exec)
 	@cp "$(EXPORT_DIR)/scripts/pycam" "$(EXPORT_DIR)/scripts/pycam-loader.py"
+	# overwrite dynamic version detection with a hardcoded string and update deb changelog
+	cd "$(EXPORT_DIR)"; \
+		echo 'VERSION = "$(VERSION)"' >pycam/version.py; \
+		debchange --newversion "$$(echo "$(VERSION)" | tr - .)" "Upstream build"; \
+		debchange --release ""
```

Comment:
Debian makes a distinction between "native" and "non-native" software, with respect to the contents of debian/changelog:

* https://www.debian.org/doc/debian-policy/ch-docs.html#s-changelogs
* https://www.debian.org/doc/debian-policy/ch-source.html#s-dpkgchangelog

I think of PyCAM as a Debian native package, by which I mean:  a package that produces Debian packages as a primary build artifact, where the Debian package is maintained and supported by the same crew of developers that maintain and support the software itself.  (This is slightly different from the conventional definition of a native package.)  Note that being a native debian package in the way I mean does not prohibit or interfere with also treating other distribution methods as primary and supported (for example #10).

I could be swayed by an argument that the top-level changelog should be updated by the top-level Makefile to add a terse git log, and the debian/changelog should be updated manually, separately.  I think the utility of two separate changelogs is low, since we maintain the packaging and the software itself in the same git repo, and I expect the debian changelog to consist nearly entirely of single-line entries like "new release".